### DISCUSSION TITLE
Gitlab runner fail

### DISCUSSION THREAD

**Discussion by Insaai** [pointfreeco/swift-perception#110] [General]
### Description

<img width="1058" alt="Снимок экрана 2024-11-15 в 11 15 50" src="https://github.com/user-attachments/assets/12676fd6-2c49-4345-be4a-eba5e7db0e05">

On Gitlab runner fails swift-perception 1.4.0.     1.3.5 is working fine.

### Checklist

- [ ] I have determined that this bug is not reproducible using Swift's observation tools. If the bug is reproducible using the `@Observable` macro or another tool from the `Observation` framework, please [file it directly with Apple](https://github.com/apple/swift/issues/new/choose).
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-perception/issues) or [discussion](https://github.com/pointfreeco/swift-perception/discussions).

### Perception version information

1.4.0

### Destination operating system

ios 13

### Xcode version information

Xcode 16

**Comment by mbrandonw** [comment]
Hi @Insaai, can you share more of the error? There isn't enough here for us to know what went wrong.

Also, you seem to be using Tuist, and so perhaps this is a problem that Tuist needs to address. I recommend opening a discussion on their repo. 

Since it isn't clear this is an issue with the library I am going to convert it to a discussion. Please feel free to continue the conversation over there!
